starter-code/server.js

```diff
@@ -55,6 +63,7 @@ app.get('/articles', function(request, response) {
 app.post('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
```

Comment:
.5 - (server.js) ln. 56 “POST: Articles for creating one new record in the database” - This is using a Method in article.js